recipes/jaws/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "jaws" %}
+{% set version = "0.2.6" %}
+{% set sha256 = "307d2e1343d8c16d4a0af836f2458339dfa57e09c0bf1559b567347f14ae3c53" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/jaws/jaws/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - pip
+    - numpy >=1.11
+  run:
+    - python
+    - pandas
+    - numpy
+    - xarray
+    - pytz
+    - netcdf4
+    - matplotlib
+    - netcdf4
+
+test:
+  imports:
+    - source
```

Comment:
Not blocking for the PR here, but I would seriously reconsider redesigning your package so that it doesn't install everything into a package named "source"

Reply:
Thank you very much conda-forge team. Apologies for silly mistakes, this is my first time building a package. I am now able to understand what is going on inside test section and why package was being installed as "source". 
I have renamed "source" directory as "jaws".